packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -512,26 +512,27 @@ export class ProjectActions {
 
       let targetTestingType: TestingType
 
+      // Get relative path from the specPath to determine which testing type from the specPattern
+      const relativeSpecPath = this.ctx.path.normalize(specPath.substring(this.ctx.currentProject.length + 1, specPath.length))
```

Comment:
I think we can just use `path.relative`, can't we?

```js
path.relative("/a/b", "/a/b/c/d.js") //=> 'c/d.js'
```

Reply:
Ahhh, yes this was what I was trying to do. Still learning some of these node best practices. Thanks!